test/Sentry.Tests/SentryClientExtensionsTests.cs

```diff
@@ -78,27 +78,6 @@ public void CaptureMessage_NullMessage_DoesNotCapturesEventWithMessage()
         Assert.Equal(default, id);
     }
 
-    [Fact]
-    public void CaptureUserFeedback_EnabledClient_CapturesUserFeedback()
```

Comment:
question: add tests for `CaptureFeedback`

We are already missing coverage for `CaptureFeedback`.
Could we, instead of deleting these tests, rewrite them to test `CaptureFeedback` instead?

Or should this be a separate PR, as it's adding coverage that is missing already.